scripts/create_tables.sql

```diff
@@ -1 +1,28 @@
 -- create doctors, patients, and appointments tables
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+CREATE TABLE patients(
+  id SERIAL PRIMARY KEY,
+  family_name TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+CREATE TABLE appointments(
+  id SERIAL PRIMARY KEY,
+  appt_date DATE,
+  appt_time TIME,
+  comment TEXT,
+  patient_id INTEGER REFERENCES patients(id),
+  doctor_id INTEGER REFERENCES doctors(id)
+);
+CREATE INDEX ON appointments(patient_id);
+CREATE INDEX ON appointments(doctor_id);
+);
```

Comment:
You have an extra closing paren here, also we don't need to create these indexes at the bottom here. You already have the references set up on the id's correctly.